icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -63,8 +64,8 @@ double testConvert(UnicodeString source, UnicodeString target, double input) {
 
 void UnitsTest::testConversionCapability() {
     struct TestCase {
-        const StringPiece source;
-        const StringPiece target;
+        const char *source;
```

Comment:
This could also be `const char *const`

Reply:
I've changed this now, since I'm touching these lines anyway, but I'm leaving it a bit inconsistent: not touching the other similar examples in this file.